dot/build_spec_integration_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/genesis"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuildFromGenesis(t *testing.T) {
+	t.Parallel()
+
+	file, err := genesis.CreateTestGenesisJSONFile(false)
+	defer os.Remove(file)
```

Comment:
You might want to rebase on `development` since this now returns a file name. The `defer os.Remove` can be removed.

Reply:
Updated.